src/garage/misc/process_samples_utils.py

```diff
@@ -0,0 +1,80 @@
+"""Utiliy functions for batch processing samples."""
```

Comment:
please don't create anything new `garage.misc` -- we are trying to delete this package.

I think you can put it in `src/garage/np/_functions.py` and then export it to `garage.np.process_samples_for_policy_gradient()` by editing `src/garage/np/__init__.py`